### PR TITLE
fix issue about pull command in repository  #3275

### PR DESCRIPTION
**Pull request by pengpengshui**

### CODE REVIEW

File: src/ui_ng/lib/src/push-image/copy-input.component.ts [hunk 1]
```diff
@@ -14,16 +14,22 @@ export const enum CopyStatus {
...
+    @Input() iconMode: boolean = false;
 
     state: CopyStatus = CopyStatus.NORMAL;
+    isHide: boolean;
 
     @Output() onCopySuccess: EventEmitter<any> = new EventEmitter<any>();
     @Output() onCopyError: EventEmitter<any> = new EventEmitter<any>();
 
+    ngOnInit(): void {
+        this.isHide = this.iconMode ?  true : false;
```
Comment (steven-zou) [inline src/ui_ng/lib/src/push-image/copy-input.component.ts]: using iconMode is enough

File: src/ui_ng/lib/src/push-image/copy-input.component.ts [hunk 2]
```diff
@@ -38,6 +44,10 @@ export class CopyInputComponent {
         this.state = CopyStatus.NORMAL;
     }
 
+    setPullCommendShow(): void {
```
Comment (steven-zou) [inline src/ui_ng/lib/src/push-image/copy-input.component.ts]: use iconMode

File: src/ui_ng/lib/src/push-image/copy-input.html.ts [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
-        <span>
+        <span  [class.hide]="isHide">
             <input type="text" class="command-input" size="{{inputSize}}" [(ngModel)]="defaultValue" #inputTarget readonly/>
         </span>
+        <span style="opacity: 0; position: absolute; z-index: -1">
+            <input type="text" size="{{inputSize}}" [(ngModel)]="defaultValue" #inputTarget1/>
+        </span>
         <span>
-            <clr-icon shape="copy" [class.is-success]="isCopied" [class.is-error]="hasCopyError" class="info-tips-icon" size="24" [ngxClipboard]="inputTarget" (cbOnSuccess)="onSuccess($event)" (cbOnError)="onError($event)"></clr-icon>
+            <clr-icon shape="copy" [class.is-success]="isCopied" [class.is-error]="hasCopyError" class="info-tips-icon" size="24" [ngxClipboard]="inputTarget1" (cbOnSuccess)="onSuccess($event)" (cbOnError)="onError($event)"></clr-icon>
```
Comment (steven-zou) [inline src/ui_ng/lib/src/push-image/copy-input.html.ts]: Try width=0px

**Comment by wy65701436** [comment]
The CI server was down today and it's back now. I restart the job just now, please see the results.

**Pull request merged**